packages/python27.rb

```diff
@@ -65,5 +65,9 @@ def self.postinstall
     #puts "Upgrading pip...".lightblue
     #system 'pip2 install --upgrade pip'
     #puts
+    FileUtils.ln_sf "#{CREW_PREFIX}/bin/python3", "#{CREW_PREFIX}/bin/python" \
+      if File.exist? "#{CREW_PREFIX}/bin/python3"
+    FileUtils.cp "#{CREW_PREFIX}/bin/pip3", "#{CREW_PREFIX}/bin/pip" \
+      if File.exist? "#{CREW_PREFIX}/bin/pip3"
```

Comment:
Should we use `FileUtils.ln_sf "#{CREW_PREFIX}/bin/pip3", "#{CREW_PREFIX}/bin/pip" \
      if File.exist? "#{CREW_PREFIX}/bin/pip3"` instead?

Reply:
Sure. Aligning with the python3 package...